layers/lldp.go

```diff
@@ -36,6 +36,11 @@ type LinkLayerDiscoveryValue struct {
 	Length uint16
 	Value  []byte
 }
+type LLDPSerializableTLV interface {
```

Comment:
Where/how is this useful?  I'm not seeing it used anywhere, and I'm hesitant to add something specifically for serializing a single layer type.

Reply:
Actually my plan was to use this for organizationally specific TLV. However, I realized we can have complete functionality without this. I will remove this.